tests/external/test_solo.py

```diff
@@ -1,12 +1,13 @@
-from scvi.data import setup_anndata, synthetic_iid
+from scvi.data import synthetic_iid
+from scvi.data._anndata import _setup_anndata
 from scvi.external import SOLO
 from scvi.model import SCVI
 
 
 def test_solo(save_path):
     n_latent = 5
     adata = synthetic_iid(run_setup_anndata=False)
-    setup_anndata(adata)
+    _setup_anndata(adata)
```

Comment:
all the tests should call the appropriate static method?

Reply:
there is no functional difference whether or not they do at this time but I can make the change